app/src/main/java/com/kimdoori/githubapp/detail/RepoDetailActivity.kt

```diff
@@ -0,0 +1,54 @@
+package com.kimdoori.githubapp.detail
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.kimdoori.githubapp.BR
+import com.kimdoori.githubapp.R
+import com.kimdoori.githubapp.common.BaseAdapter
+import com.kimdoori.githubapp.databinding.ActivityRepoDetailBinding
+import com.kimdoori.githubapp.domain.model.PullRequestModel
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class RepoDetailActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityRepoDetailBinding
+    private val viewModel: RepoDetailViewModelImpl by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setupBinding()
+        setupRecyclerView()
+        showContent()
+    }
+
+    private fun setupBinding() {
+        binding = DataBindingUtil
+            .setContentView<ActivityRepoDetailBinding>(this, R.layout.activity_repo_detail)
+            .apply {
+                lifecycleOwner = this@RepoDetailActivity
+                viewModel = this@RepoDetailActivity.viewModel
+            }
+    }
+
+    private fun setupRecyclerView() {
+        binding.repoDetailPullrequestRecyclerview.adapter = BaseAdapter<PullRequestModel>(
+            layoutResourceId = R.layout.item_pullrequest,
+            bindingVariableId = BR.item,
+        )
+    }
+
+    private fun showContent() {
+        val userName = intent.getStringExtra(EXTRA_USER_NAME)
+        val repoName = intent.getStringExtra(EXTRA_REPO_NAME)
+        if (userName == null || repoName == null) return
```

Comment:
```suggestion
        val userName = intent.getStringExtra(EXTRA_USER_NAME) ?: return
        val repoName = intent.getStringExtra(EXTRA_REPO_NAME) ?: return
```
이렇게도 사용할 수 있을 것 같습니다 😄

Reply:
제안 감사합니다! https://github.com/kimdoori/GitHubApp/pull/22/commits/8efbddf4593d082ae2dbc46a356a3b1290728b8d